### PR TITLE
(feat) remove the check for order_level_amount > 0

### PR DESCRIPTION
**Pull request by vic-en**
**Before submitting this PR, please make sure**:
- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/", etc)

**Optional**:
- Related Github issue: #1669 
- Related Clubhouse Story:

**A description of the changes proposed in the pull request**:
Removed check for minimum order_level_amount in PMM.

### CODE REVIEW

File: hummingbot/strategy/pure_market_making/pure_market_making_config_map.py [hunk 1]
```diff
@@ -174,7 +174,7 @@ def exchange_on_validated(value: str):
                          "additional order? >>> ",
```
Comment (Nullably) [inline hummingbot/strategy/pure_market_making/pure_market_making_config_map.py:174]: I think we should change to the prompt message a bit so user knows that decreasing order size is also an option. And we'll need to update doc and the template parameter description as well.

Comment (vic-en) [reply hummingbot/strategy/pure_market_making/pure_market_making_config_map.py:174]: Done!

**Comment by mifeng** [comment]
Since this is an enhancement, I think we should merge this fix in after release

**Comment by vic-en** [comment]
> 
> 
> Sorry @vic-en I should have triaged this story more carefully before adding this to the Public Roadmap.
> 
> This story is a good one, but I think the change should be in `order_level_amount`, not in `order_levels`. To allow users to decrease order sizes at higher levels, it's more intuitive to enter negative values for `order_level_amount`, so that the sizes decrease at higher order levels. Negative order levels are confusing.
> 
> Can you change this enhancement so that it adjusts the level amount instead?

@mifeng , The initial change was for `order_level_amount`.

**Comment by mifeng** [comment]
@vic-en OK thanks - I'll review and test properly this time

**Pull request merged**